Packs/DeCYFIR/Integrations/decyfir/decyfir_test.py

```diff
@@ -0,0 +1,384 @@
+from datetime import datetime, timedelta
+import json
+import io
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+def test_say_hello():
```

Comment:
Please change the test name to match the integration :)

Reply:
ok